### PR TITLE
convert showheader's status changer to vue

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Shouldn't need to update snatch selection as it doesn't use this even though it uses `showheader.mako`.

### CODE REVIEW

File: themes-default/slim/src/components/display-show.vue [hunk 1]
```diff
@@ -47,7 +48,8 @@ export default {
     computed: {
         ...mapState({
             shows: state => state.shows.shows,
-            indexerConfig: state => state.config.indexers.config.indexers
+            indexerConfig: state => state.config.indexers.config.indexers,
+            statusStrings: state => flip(state.statuses.strings)
```
Comment (sharkykh) [inline themes-default/slim/src/components/display-show.vue]: just a suggestion, isn't this simpler?
https://github.com/pymedusa/Medusa/blob/7597778ce0c19be4af039877127342eba7e4c316/themes-default/slim/src/components/add-show-options.vue#L239-L248

File: themes-default/slim/src/components/display-show.vue [hunk 2]
```diff
@@ -89,6 +91,31 @@ export default {
             const id = show.id[show.indexer];
             const indexerUrl = indexerConfig[show.indexer].showUrl;
             return `${indexerUrl}${id}`;
+        },
+        statuses() {
+            const { statuses, config } = this.$store.state;
```
Comment (sharkykh) [inline themes-default/slim/src/components/display-show.vue]: you're destructuring from `this.$store.state`, but I think you meant to use `mapState` for these

Comment (OmgImAlexis) [reply themes-default/slim/src/components/display-show.vue]: Nope, read the function.

Comment (p0psicles) [reply themes-default/slim/src/components/display-show.vue]: I don't understand this. A little bit more explanation would be nice.

File: themes-default/slim/package.json [hunk 3]
```diff
@@ -60,6 +60,7 @@
     "imagemin-pngquant": "6.0.0",
     "is-visible": "2.2.0",
     "jquery": "3.3.1",
+    "just-flip-object": "1.1.0",
```
Comment (sharkykh) [inline themes-default/slim/package.json]: https://github.com/sharkykh/Medusa/commits/feature/vueify-showheader-statuschanger-updated

**Comment by sharkykh** [comment]
Replaced by #6189

**Pull request closed without merging**